script/install.sh

```diff
@@ -619,7 +619,7 @@ uninstall_compiler() {
         fatal "$1 is not installed in $path"
     fi
     log "Removing $path/$1"
-    rm -rf "$path/$1"
+    rm -rf "${path:?}/$1"
```

Comment:
With `set -u`, `$var` will error only if it's unset, not empty. `${var:?}` will always error if `var` is empty (or unset).